MapboxMobileEvents/MMEMetrics.h

```diff
@@ -9,6 +9,9 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) NSUInteger totalDataTransfer;
```

Comment:
Suggest making `readonly`, renaming `totalBytesTransferred` and computing from `cellBytesTransfered + wifiBytesTransfered`